<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1559040600622" stop="1559040618368">
  <name>test_login</name>
  <labels/>
  <test-cases>
    <test-case start="1559040600622" status="broken" stop="1559040618368">
      <description>
        正向测试用例
        :param case_num: 用例编号
        :param account: 用户名
        :param passwd: 密码
        :param exp_data: 预期结果
        :return:
        </description>
      <name>TestLogin.test_login_suc</name>
      <failure>
        <message>FixtureLookupError: ('account', &lt;FixtureRequest for &lt;Function 'test_login_suc'&gt;&gt;)</message>
        <stack-trace>file C:\Users\Administrator.DESKTOP-6R85566\PycharmProjects\sh_app09_test\scripts\test_login.py, line 42
      def test_login_suc(self, account, passwd, exp_data):
E       fixture 'account' not found
&gt;       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, metadata, monkeypatch, pytestconfig, record_property, record_xml_attribute, record_xml_property, recwarn, tmpdir, tmpdir_factory
&gt;       use 'pytest --fixtures [testpath]' for help on them.

C:\Users\Administrator.DESKTOP-6R85566\PycharmProjects\sh_app09_test\scripts\test_login.py:42</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="17344-MainThread"/>
        <label name="host" value="lcg"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
